checkov/terraform/checks/resource/azure/StorageBlobServiceContainerPrivateAccess.py

```diff
@@ -0,0 +1,19 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class StorageBlobServiceContainerPrivateAccess(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that 'Public access level' is set to Private for blob containers"
+        id = "CKV_AZURE_34"
+        supported_resources = ['azurerm_storage_container']
+        categories = [CheckCategories.LOGGING]
```

Comment:
@arielkru This is `NETWORKING` I believe

Reply:
Right